core/sis-feature/src/test/java/org/apache/sis/feature/builder/FeatureTypeBuilderTest.java

```diff
@@ -344,4 +346,98 @@ private static void assertPropertyEquals(final String name, final Class<?> value
         }
         assertEquals("valueClass", valueClass, ((DefaultAttributeType<?>) property).getValueClass());
     }
+
+    @Test
+    public void testToStringInternalThrowsNullPointerException() {
```

Comment:
I'm not sure what is the purpose of verifying that `featureTypeBuilder.toStringInternal` throws a `NullPointerException`? This is internal mechanic that could change at any time. For example I didn't checked yet but maybe the `NullPointerException` is rather a bug than something we want to enforce?

Reply:
I just wanted to create a regression test that holds the current behavior.
However, reading your comment this obviously is a bug.
I am going to set the test to '@Ignore' with the comment that it is supposed to be a bug.
Furthermore, I'll going to repair it in a new future branch/pull request.